aw_transform/merge_events_by_keys.py

```diff
@@ -13,22 +13,14 @@ def merge_events_by_keys(events, keys) -> List[Event]:
         return events
     merged_events = {}  # type: Dict[str, Event]
     for event in events:
-        summed_key = ""
-        for key in keys:
-            if key in event.data:
-                summed_key = summed_key + "." + event["data"][key]
+        summed_key = "/".join(event['data'][key] for key in keys)
```

Comment:
This would be worth keeping.

Reply:
Looks fine, probably more efficient as well.